mypy/stubdoc.py

```diff
@@ -247,6 +247,14 @@ def infer_arg_sig_from_docstring(docstr: str) -> List[ArgSig]:
     return []
 
 
+def infer_ret_type_sig_from_docstring(docstr: str) -> Optional[str]:
```

Comment:
What about renaming the function to indicate that this parses a return type using an alternative docstring syntax (`infer_arg_sig_from_docstring` is also similar and could be renamed)? Maybe add an `_alt` suffix or something. The main different seems to be that this syntax is missing the function name in the docstring.

Reply:
`infer_ret_type_sig_from_partial_docstring`? `infer_ret_type_sig_from_anon_docstring`?